packages/ods-react/src/components/combobox/src/components/combobox-control/ComboboxControl.tsx

```diff
@@ -0,0 +1,77 @@
+import { Combobox as VendorCombobox, useComboboxContext } from '@ark-ui/react/combobox';
+import classNames from 'classnames';
+import { type ComponentPropsWithRef, type FC, type JSX, forwardRef, useContext } from 'react';
+import { Input } from '../../../../input/src';
+import { ComboboxContext } from '../../context/combobox';
+import style from './comboboxControl.module.scss';
+
+interface ComboboxControlProp extends ComponentPropsWithRef<'button'> {
+  clearable?: boolean;
+  loading?: boolean;
+  placeholder?: string;
+}
+
+const ComboboxControl: FC<ComboboxControlProp> = forwardRef(({
+  className,
+  clearable = false,
+  loading = false,
+  placeholder,
+  ...props
+}, ref): JSX.Element | null => {
+  const context = useComboboxContext();
+  const { placement } = useContext(ComboboxContext);
+  if (!context) {
```

Comment:
if we start to handle this, we should use a generic way to avoid defining it every time